plugins/UFM_NDT_Plugin/ufm_sim_web_service/topo_diff/ndt_infra.py

```diff
@@ -169,6 +170,16 @@ def run_ibdiagnet_verification_command():
     status, cmd_output = execute_generic_command(IBDIAGNET_PORT_VERIFICATION_COMMAND)
     return status
 
+def normalize_port_guid_length(port_guid):
+    '''
+    Normalize port guid len - in need add prefix 0
+    :param port_guid:
+    '''
+    port_guid_len = len(port_guid)
+    if port_guid_len < 18: # in case and we have guid that is short - need to add zero
+        port_guid = "".join(["0x", "%s"%("0"*(18-port_guid_len)), port_guid[2:]])
```

Comment:
f strings
```suggestion
        port_guid = "".join(["0x", f"{"0" * (18-port_guid_len)}"), port_guid[2:]])
```

Reply:
It is the same. And it is additional location of such command - so to be aligned to the code.